runtime/runtime.go

```diff
@@ -31,32 +34,8 @@ import (
 	"github.com/open-policy-agent/opa/storage/inmem"
 	"github.com/open-policy-agent/opa/util"
 	"github.com/open-policy-agent/opa/version"
-	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
 )
 
-var (
-	registeredPlugins    []pluginFactory
-	registeredPluginsMux sync.Mutex
-)
-
-// RegisterPlugin registers a plugin with the runtime package. When a Runtime
-// is created, the factory functions will be called.
-func RegisterPlugin(name string, factory func(m *plugins.Manager, config []byte) (plugins.Plugin, error)) {
```

Comment:
This is a breaking change. Unless there's a compelling reason to move the this function into the plugins package, it should be left here.

Reply:
There's a good reason -- see commit ffea2127eba537173a8e4e3f9be8a658189d5ed0

Reply:
A workaround would be to create a function in the Plugins package that calls the runtime version, but that seemed messy to me.

Reply:
If we move `RegisterSharedObjectsFromDir` out of the `runtime` package (e.g., into `cmd` where it can be unexported) would that fix this?